ICA/auto_Granduer_ICA.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import json
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date,datetime,timedelta
+from collections import Counter
+
+# This function uses the ica CLI tool to return info from ICA
+def icav_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('\t', ' ') for x in tmp]
+    return tmp
+
+# While loop is infinite will keep script running as long as it doesn't error out.
+go=1
+while go == 1:
+    print('Looking for analyses that need to start!')
+    #This block set the icav2 tool to correct project in use
+    bashCommand="icav2 projects enter Testing"
+    process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+    process.communicate()
+
+    #This  block collects the information of data and runs on ICA
```

Comment:
It's this portion of this file that is holding up this PR. I hesitate to have dual while loops running.

Moving forward we should
1. Remove the loop and have the bash script run this command once the fastq files have finished downloading
2. This loop needs to also search through basespace for new run, download fastq files, and then continue on

I feel bad that this is holding this PR up. Should we remove this file from the PR and at least get the mycosnp stuff going?